### ISSUE TITLE
Drop Feature

### ISSUE THREAD

**Issue by imtvwy** [UBC-MDS/Giant_Pumpkins_Weight_Prediction#17]
I have a hard time to clean the 'mother_seed' and 'father_pollinator' column.  The data simply doesn't really conform the standard we found in the website 'weight seed year'.  There are also a lot of typos even for the NA values.  

After spending the whole night trying to figure that out, I'd suggest to skip these 2 columns for our project at this time.  
Justification:
1. In our EDA, we already see evidence that other features (ott, country, city, grower) have relationship with target
2. Though we think 'genetic' data may affect the weight intuitively, we can start by using a subset of features first to see how our model performs
3.  Always good to keep our project simple and try on the project workflow, rather than spending time in tidying data

Please comment!

**Comment by mahsasarafrazi** [comment]
Yeah, I have worked with the raw data and I agree, we can drop both columns and also value column which is totally Nan

**Comment by shivajena** [comment]
I also agree, I tried as well but I will try once more on the tidy part on parent seeds features till tonight and if successful, and if i find anything interesting, will be happy to share. For now, have to sort the ML script issue with the TAs
